File: AcademicWebApp/src/main/java/com/example/AcademicWebApp/Models/Faculty.java
package com.example.AcademicWebApp.Models;

import lombok.*;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;

@Data
@Entity(name = "faculty")
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Faculty {

    @Id
    @Column(name="fid")
    private int fid;

    @Column(name="name")
    private String name;

    @Column(name="noyears")
    private int noyears;

    @Column(name="chief")
    private String chief;

    @Override
    public String toString() {
        return "Faculty{" +
                "fid=" + fid +
                ", name='" + name + '\'' +
                ", noYears=" + noyears +
                ", chief='" + chief + '\'' +
                '}';
    }
}

File: DataGenerator/domain/faculty.py


class Faculty:

    def __init__(self, id, name, section, noYears, chief):
        self.__id = id
        self.__name = name
        self.__section = section
        self.__noYears = noYears
        self.__chief = chief

    @property
    def id(self):
        return self.__id

    @property
    def name(self):
        return self.__name

    @property
    def section(self):
        return self.__section

    @property
    def noYears(self):
        return self.__noYears

    @property
    def chief(self):
        return self.__chief

    def __str__(self):
        return str(self.__id) + ", '" + self.__name + "-" + self.__section + "', " + str(self.__noYears) + ", '" + self.__chief + "'"
